en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide-docker-with-apim.md

```diff
@@ -76,14 +88,36 @@ the changes done to the API into Choreo Connect.
     **JWT tokens**:
     As the application is self-contained (JWT), **copy the generated access token** before proceeding to the next step.
 
-### Step 6 - Invoke the API via Choreo Connect
+### Step 5 - Invoke the API via Choreo Connect
 
-Use the below curl command to invoke the `/menu` resource of the PizzaShackAPI
+Follow the instructions below to invoke the previously created API with the generated keys.
 
-``` java
-curl -k -X GET "https://localhost:9095/pizzashack/1.0.0/menu" -H "accept: application/json" -H "Authorization: Bearer <COPIED_TOKEN>"
-```
+1. Click **Try Out** on the left menu bar.
 
-!!! info
-    Note that the port 9095 in the above cURL command is the Choreo Connect port. Hence, it can be inferred that you are invoking the API
-    via Choreo Connect.
+     The resources of the API will be listed.
+
+2. Paste the access token that you previously copied in the **Access Token** field.
+
+    [![Paste the access token]({{base_path}}/assets/img/deploy/mgw/invoke-api.png)]({{base_path}}/assets/img/deploy/mgw/invoke-api.png)
+
+3. **If this is the first time you are using the API test console** from your browser,  open a new tab and navigate to the [https://localhost:9095/](https://localhost:9095/) URL.
+
+     This will prompt your browser to accept the certificate used by the Choreo Connect. This is required because by default the Choreo Connect uses a self-signed certificate that is not trusted by web browsers.
```

Comment:
```suggestion
     This will prompt your browser to accept the certificate used by Choreo Connect. This is required because, by default, Choreo Connect uses a self-signed certificate that is not trusted by web browsers.
```